recipe-server/normandy/recipes/tests/__init__.py

```diff
@@ -108,6 +106,21 @@ def locales(self, create, extracted, **kwargs):
             for locale in extracted:
                 self.locales.add(locale)
 
+    @factory.post_generation
+    def enabled(self, create, extracted, **kwargs):
+        if extracted:
+            approval = ApprovalRequestFactory(revision=self.latest_revision)
+            approval.approve(UserFactory(), 'r+')
```

Comment:
I think I'd be less surprised if this were in a helper function instead of being based off a factory attribute. The helper would be more explicit about creating the user and could allow you to override it more easily.